### PR TITLE
Remove hard coded values

### PR DESCRIPTION
**Pull request by Jayendra88**
**What problem is this solving?**
Remove hard coded values for `baseLiveAgentContentURL` and `baseLiveAgentURL` 

**How should this be manually tested?**

Admin app should show input fields to get a value from settings.

### CODE REVIEW

**Comment by Jayendra88** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually disappointed

**Comment by Jayendra88** [comment]
> Beep boop 
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document thinking
> * [ ]  I'll do it later disappointed
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated tadatada

File: pixel/body.html [hunk 1]
```diff
@@ -63,11 +63,10 @@
         '{{settings.organizationId}}',
         '{{settings.accountName}}',
         {
-          baseLiveAgentContentURL:
-            'https://c.la1-c2cs-ia2.salesforceliveagent.com/content',
+          baseLiveAgentContentURL: window.decodeURIComponent('{{settings.baseLiveAgentContentURL}}'),
```
Comment (klzns) [inline pixel/body.html]: I think we should have the previous values as default values, otherwise other stores that might use this will break.

**Review by klzns — CHANGES_REQUESTED** [review]

File: pixel/body.html (hunk 1 above)
Comment (Jayendra88) [reply pixel/body.html]: Fixed

**Comment by tomymehdi** [comment]
This might break already working stores. Please increase the major on the manifest se we can avoid the long process of dicovery of a minor / patch

**Comment by Jayendra88** [comment]
Upgrade `1.0.0` to `2.0.0` in `manifest.json`

File: pixel/body.html [hunk 2]
```diff
@@ -56,18 +56,25 @@
       embedded_svc.settings.enabledFeatures = JSON.parse(enabledFeatures)
       embedded_svc.settings.entryFeature = '{{settings.entryFeature}}'
 
+      var baseLiveAgentContentURL = '{{settings.baseLiveAgentContentURL}}' == ''
```
Comment (tomymehdi) [inline pixel/body.html]: Let's remove this conditional. v2 won't affect previous implementations

File: pixel/body.html [hunk 3]
```diff
@@ -56,18 +56,25 @@
...
+        ? 'https://c.la1-c2cs-ia2.salesforceliveagent.com/content'
+        : window.decodeURIComponent('{{settings.baseLiveAgentContentURL}}')
+
+      var baseLiveAgentURL = '{{settings.baseLiveAgentURL}}' == ''
```
Comment (tomymehdi) [inline pixel/body.html]: Let's remove this conditional. v2 won't affect previous implementations

**Review by tomymehdi — CHANGES_REQUESTED** [review]

File: pixel/body.html (hunk 2 above)
Comment (Jayendra88) [reply pixel/body.html]: fixed

File: pixel/body.html (hunk 3 above)
Comment (Jayendra88) [reply pixel/body.html]: fixed

**Review by tomymehdi — APPROVED** [review]

**Pull request merged**